db/dataset_eval.py

```diff
@@ -11,12 +11,14 @@
 STATUS_RUNNING = "running"
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
-
+# Eval locations are defined in `eval_location_type` type. See schema definition.
+EVAL_LOCAL = "local"
+EVAL_REMOTE = "remote"
 # Filter types are defined in `eval_filter_type` type. See schema definition.
 FILTER_ARTIST = "artist"
 
 
-def evaluate_dataset(dataset_id, normalize, filter_type=None):
+def evaluate_dataset(dataset_id, normalize, filter_type=None, eval_location=EVAL_LOCAL):
```

Comment:
I did that because if this functions is might be called somewhere else which I might not have noticed we wont have to change that. It can take a default value on its on, and wherever we need to give it a value of our choice we are doing that. However this might be against good software practice, so if you think we need to change that I can do that.